phpunit.xml.dist

```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
-         backupGlobals="false"
-         colors="true"
-         verbose="true"
-         beStrictAboutOutputDuringTests="true"
-         beStrictAboutTestsThatDoNotTestAnything="true"
```

Comment:
Why was this removed?

Reply:
I've tried explained on the commit message (cb7f39d) but the reason is that we were just repeating the default configuration, which is kind of redundant.